Assets/Scripts/Interactables/GiantFanController.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEngine;
+
+public class GiantFanController : MonoBehaviour
+{
+    [SerializeField]
+    private float airForce = 35f;
+
+    private void OnTriggerStay(Collider other)
+    {
+        if (other.gameObject.TryGetComponent<Rigidbody>(out Rigidbody rigidbodyPlayer))
+        {
+            rigidbodyPlayer = other.gameObject.GetComponent<Rigidbody>();
```

Comment:
No need to find this again, you already have the rigidbody right there! `out` parameters are assigned/changed inside the method they're passed to.